cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml

```diff
@@ -1,36 +1,36 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: monitoring-influxdb-grafana-v2
+  name: monitoring-influxdb-grafana-v3
   namespace: kube-system
-  labels: 
+  labels:
     k8s-app: influxGrafana
-    version: v2
+    version: v3
     kubernetes.io/cluster-service: "true"
-spec: 
+spec:
   replicas: 1
-  selector: 
+  selector:
     k8s-app: influxGrafana
-    version: v2
-  template: 
-    metadata: 
-      labels: 
+    version: v3
+  template:
+    metadata:
+      labels:
         k8s-app: influxGrafana
-        version: v2
+        version: v3
         kubernetes.io/cluster-service: "true"
-    spec: 
-      containers: 
-        - image: gcr.io/google_containers/heapster_influxdb:v0.4
+    spec:
+      containers:
+        - image: gcr.io/google_containers/heapster_influxdb:v0.6
           name: influxdb
           resources:
             # keep request = limit to keep this container in guaranteed class
             limits:
-              cpu: 100m
-              memory: 200Mi
+              cpu: 1000m
+              memory: 600Mi
             requests:
-              cpu: 100m
-              memory: 200Mi
-          ports: 
+              cpu: 1000m
+              memory: 600Mi
```

Comment:
This change might affect our test cells since 600Mi is a large request for small VMs. 
The default config is not meant to work for everyone. Can you update your InfluxDB limits manually in your clusters? For better performance, one might want to give InfluxDB a few more cores and memory.